webClient/src/app/browser-panel/browser-panel.component.ts

```diff
@@ -183,9 +197,16 @@ export class BrowserPanelComponent implements OnInit {
         const filename = tokens[tokens.length - 1];
         this.log.debug('saveAs filename=' + filename);
         const a = document.createElement('a');
+        this.log.debug('downloading from uri', uri, 'with path ',this.selectedPath);
```

Comment:
Additionally, and I haven't checked if other parts of the code are violating this, but we should change all instances of "console.log" where we can to use the Zowe logger
https://github.com/zowe/zlux/wiki/Logging

Fairly simple to use, logging translation IDs are not required yet, we just want to use the component